sample/admin.py

```diff
@@ -7,10 +6,12 @@
 
 from sample import models
 
+
 class SampleModelAdmin(admin.ModelAdmin):
     formfield_overrides = {
         AddressField: {'widget': GoogleMapsAddressWidget},
```

Comment:
It is useful to show how to adjust the map type - but I wonder if adding instructions in the readme might be better.  My concern is it might be confusing to have the default Map Type be `hybrid` on the javascript side, but then the sample app people see first would show a `roadmap`.

Interested in your thoughts. Either way if it is here it should be on the address field.